evm_loader/program/src/entrypoint.rs

```diff
@@ -306,12 +313,14 @@ fn do_call<'a>(
 {
     info!("do_call");
     let account_info_iter = &mut accounts.iter();
+    let myself_info = next_account_info(account_info_iter)?;
```

Comment:
This line break lines 327-328. I suggest use `program_id` where it needs. For correct transaction user should only add program account info in any place in middle of list.

Reply:
@s-medvedev no, this line doesn't break anything, because in 322 we splitting it out and in 323 passing accounts without it.

Reply:
If you remove this entry from the list, you will most likely have problems with `envoke`.